runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1750,60 +1839,164 @@ static void buildInterfaceCall(TR::CodeGenerator *cg, TR::Node *callNode, TR::Re
+            /* Generating instructions after debug counter instructions. */
+            TR::Instruction *slot2DoneLabelInstr = generateLabelInstruction(cg, TR::InstOpCode::label, callNode, slot2DoneLabel);
+            if (debugObj)
+               {
+               debugObj->addInstructionComment(branchToSlot2DoneLabelInstr, "Jumps to slot2DoneLabel");
+               debugObj->addInstructionComment(slot2DoneLabelInstr, "slot2DoneLabel");
+               }
+            }
+         }
+      }
+
+   generateCompareInstruction(cg, callNode, vftReg, tmpReg, true);
+
+   TR::SymbolReference  *methodSymRef = callNode->getSymbolReference();
+   TR_ResolvedMethod    *owningMethod = methodSymRef->getOwningMethod(comp);
+   uintptr_t            itableIndex;
+   TR_OpaqueClassBlock  *interfaceClassOfMethod = owningMethod->getResolvedInterfaceMethod(methodSymRef->getCPIndex(), &itableIndex);
+
+   TR::Instruction *gcPoint;
+   if (useLastITableCache && (interfaceClassOfMethod != NULL))
+      {
+      TR_J9VMBase *fej9 = cg->fej9();
+
+      TR::LabelSymbol *secondBranchAddressCacheSlotLabel = ifcSnippet->getSecondBranchAddressCacheSlotLabel();
+      generateTrg1ImmSymInstruction(cg, TR::InstOpCode::ldrx, callNode, tmpReg, 0, secondBranchAddressCacheSlotLabel);
+      generateConditionalBranchInstruction(cg, TR::InstOpCode::b_cond, callNode, hitLabel, TR::CC_EQ);
+
+      generateTrg1MemInstruction(cg, TR::InstOpCode::ldrimmx, callNode, tmpReg, TR::MemoryReference::createWithDisplacement(cg, vftReg, fej9->getOffsetOfLastITableFromClassField()));
+      TR::LabelSymbol *interfacedClassSlotLabel = ifcSnippet->getInterfaceClassSlotLabel();
+      generateTrg1ImmSymInstruction(cg, TR::InstOpCode::ldrx, callNode, x9Reg, 0, interfacedClassSlotLabel);
+      generateTrg1MemInstruction(cg, TR::InstOpCode::ldrimmx, callNode, tmp2Reg, TR::MemoryReference::createWithDisplacement(cg, tmpReg, fej9->getOffsetOfInterfaceClassFromITableField()));
+      generateCompareInstruction(cg, callNode, x9Reg, tmp2Reg, true);
+
+      if (comp->getOptions()->enableDebugCounters())
+         {
+         TR::Instruction *prevCursor1 = cg->getAppendInstruction();
+         /* Record if lastITable cache hit */
+         TR::Instruction *cursor1 = cg->generateDebugCounter(TR::DebugCounter::debugCounterName(comp, "cg.callInterface/(%s)/%s/%d/%d/dynamicPIC/lastITable",
                                                                                              comp->signature(),
                                                                                              comp->getHotnessName(),
                                                                                              callNode->getByteCodeInfo().getCallerIndex(),
                                                                                              callNode->getByteCodeInfo().getByteCodeIndex()), *srm);
```

Comment:
Indentation of these arguments is different from that for `generateDebugCounter()` for `cursor2` below.

Reply:
Fixed the indentation.